Gruntfile.js

```diff
@@ -144,7 +144,8 @@ module.exports = function (grunt) {
                     },
                     transform: [
                         fileMapTransform,
-                        ['babelify']
+                        ['babelify'],
+                        ['brfs']
```

Comment:
our tests run in a browser where we don't have access to disk, so we have to inline everything. `require`s are automatically bundled by browserify but for `fs.readFileSync` we needed another transform